.github/renovate.json5

```diff
@@ -85,5 +85,5 @@
     ],
     "platformAutomerge": true,
   }],
-  "labels": ["auto-approve"],
+  "labels": ["auto-approve", "build-builder-image", "rebuild-test-container"],
```

Comment:
This change essentially means that we will also add those labels to renovate PRs that point to master, correct? Not a huge deal, the CI on those PRs will just be extremely long.